docs/viplab3.0/computation_result.md

```diff
@@ -75,8 +102,9 @@ wrapper "Result" around the following
 
 |Key            |Value Type |Opt / Must |Description |Comment |
 |---------------|-----------|-----------|------------|--------|
-|ID                   |string     |must | unique Result ID created by CC (automatically) | For tracking and debugging purposes. |
-|comment              |string     |opt              | comment from CC | |
+|identifier     |string (uuid) |must       | unique Result ID created by CC (automatically) | For tracking and debugging purposes. |
+|version | string | opt | version of the json specification | should be given for backwards compatibility |
```

Comment:
why is the version optional? What is the default version?

Reply:
fixed